packages/renderer/src/lib/preferences/PreferencesResourcesRendering.svelte

```diff
@@ -259,6 +260,9 @@ async function startConnectionProvider(
           </div>
           <!-- providers columns -->
           <div class="grow flex flex-wrap divide-gray-600 ml-2">
+            <PreferencesConnectionsEmptyRendering 
+              message="{provider.kubernetesProviderEmptyConnectionsViewMessage}"
+              hidden="{provider.containerConnections.length >0 || provider.kubernetesConnections.length >0}"/>
```

Comment:
This part seems a bit odd, I think the formatting is a bit off / logic behind this.

Otherwise, the PR LGTM. You'll have to run `yarn format:fix`, rebase and push to fix the errors.

Reply:
this should not be here, it should be in UI part